elliottlib/cli/find_bugs_sweep_cli.py

```diff
@@ -308,30 +308,30 @@ def categorize_bugs_by_type(bugs: List[Bug], advisory_id_map: Dict[str, int], ma
             continue
         attached_builds = errata.get_advisory_nvrs(advisory)
         packages = list(attached_builds.keys())
-        exception_packages = []
         if kind == 'image':
             # golang builder is a special tracker component
             # which applies to all our golang images
-            exception_packages.append(constants.GOLANG_BUILDER_CVE_COMPONENT)
-
-        if kind == 'microshift':
-            # microshift is special since it has a separate advisory, and it's build is attached
-            # after payload is promoted. So do not pre-emptively complain
-            exception_packages.append('microshift')
+            packages.append(constants.GOLANG_BUILDER_CVE_COMPONENT)
```

Comment:
Why is `exception_packages` removed? I think it is better to keep it and make our logs clear about it is an exception.

Reply:
pr updated, keep golang bug in exception